scripts/lochness_create_template.py

```diff
@@ -305,6 +305,9 @@ def create_config_template(config_loc: Path, args: object) -> None:
 mindlamp_days_to_pull: 100
 pii_table: {args.pii_csv}
 lochness_sync_history_csv: {args.lochness_sync_history_csv}
+days_to_keep: 100
+removed_df_loc: {args.outdir}/PHOENIX/removed_files.csv
+removed_phoenix_root: {args.outdir}/track_removed_files_PHOENIX
```

Comment:
Didn't Sylvain say 14 days for default?

Reply:
Yes- about two weeks. I don't want to start deleting files yet, this is still at its testing phase.